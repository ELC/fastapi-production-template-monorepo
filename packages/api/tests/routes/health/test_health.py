from fastapi import status
from fastapi.testclient import TestClient
from syrupy.assertion import SnapshotAssertion


def test_health_endpoint(client: TestClient, snapshot: SnapshotAssertion) -> None:
    response = client.get("/health")

    assert response.status_code == status.HTTP_200_OK
    assert response.headers["Content-Type"] == "application/json"
    assert response.json() == snapshot
