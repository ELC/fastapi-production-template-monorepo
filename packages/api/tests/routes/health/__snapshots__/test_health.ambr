# serializer version: 1
# name: test_health_endpoint
  dict({
    'message': 'The API is running smoothly.',
    'status': 'healthy',
  })
# ---
